recipes/firefox-unbranded/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+APP_DIR=$PREFIX/bin/FirefoxApp
+LAUNCH_SCRIPT=$PREFIX/bin/firefox
+
+mkdir -p $APP_DIR
+
+if [[ $(uname) == Linux ]]; then
+  mv * $APP_DIR
+  BIN_LOCATION=$APP_DIR/firefox
+fi
+
+# Write launch script and make executable
+cat <<EOF >$LAUNCH_SCRIPT
+#!/bin/bash
+$BIN_LOCATION "\$@"
```

Comment:
If this is all that this script does, why not make a symlink from `BIN_LOCATION` to `$PREFIX/bin/firefox` ?

Reply:
We went through a number of iterations when building the main firefox recipe. (https://github.com/conda-forge/staged-recipes/pull/9234)

This suggestion wasn't one of them, but it seems like a good one. We were following a pattern from a number of other recipes. But your suggestion makes good sense to me. And I think it would also work on OSX.

At the moment, my main motivation for keeping it like this is to maintain the highest possible degree of similarity between this and the standard firefox-feedstock to keep maintenance as simple as possible.

@ocefpaf would appreciate your second opinion.

Reply:
Separately, this is not the problem that's causing this to not build in docker. Any help on that issue would be greatly appreciated @isuruf.